gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc

```diff
@@ -53,13 +59,13 @@ To release a custom policy, run the `release` command from the policy's root fol
 make release
 ----
 
-After releasing the asset in Exchange, the asset's GAV appears, similar to the following:
+After releasing the asset in Exchange, the `DEV` label is removed. The asset's GAV appears similar to the following:
```

Comment:
The label is not removed from previously published assets. This is a different asset that is being uploaded.